src/twisted/protocols/ftp.py

```diff
@@ -736,10 +739,20 @@ class FTP(basic.LineReceiver, policies.TimeoutMixin, object):
     passivePortRange = xrange(0, 1)
 
     listenFactory = reactor.listenTCP
+    _encoding = 'latin-1'
 
     def reply(self, key, *args):
         msg = RESPONSE[key] % args
-        self.sendLine(msg)
+        self.sendEncodedLine(msg)
+
+
+    def sendEncodedLine(self, line):
+        """
+        (Private) Encodes and sends a line
```

Comment:
IIRC `sendLine` is a method on the superclass; I didn't want to mess with that one, since it'd prevent sending unencoded lines. I'll take another look at this, thanks.